Library/Homebrew/formula_installer.rb

```diff
@@ -101,7 +101,7 @@ def pour_bottle?(install_bottle_options = { warn: false })
       return false
     end
 
-    unless bottle.compatible_cellar?
+    if bottle && !bottle.compatible_cellar?
```

Comment:
`unless bottle&.compatible_cellar?`

Reply:
I didn't know about the `&.` operator! Fantastic!

Reply:
Will be enforced by `brew style` from https://github.com/Homebrew/homebrew-portable/pull/50. Wasn't supported in Ruby 2.0.

Reply:
`formula.bottle` is `nil` when `bottle.compatible_cellar?` is false.
Using `formula.bottle_specification.compatible_cellar?` rather
than `formula.bottle.compatible_cellar?` avoids this problem